internal/testrunner/runners/system/runner.go

```diff
@@ -478,6 +478,11 @@ func (r *runner) runTest(config *testConfig, ctxt servicedeployer.ServiceContext
 		}
 	}
 
+	// Check Hit Count within docs, if 0 then it has not been specified
+	if assertion_pass, message := assertHitCount(config.Assert.HitCount, docs); !assertion_pass {
```

Comment:
Go naming conventions are `mixedCaps`. https://go.dev/doc/effective_go#mixed-caps

```suggestion
	if assertionPass, message := assertHitCount(config.Assert.HitCount, docs); !assertionPass {
```